swap/log.go

```diff
@@ -0,0 +1,114 @@
+package swap
+
+import (
+	l "github.com/ethereum/go-ethereum/log"
+)
+
+const (
+	// CallDepth is set to 1 in order to influence to reported line number of
+	// the log message with 1 skipped stack frame of calling l.Output()
+	CallDepth = 1
+	//DefaultAction is the default action filter for SwapLogs
+	DefaultAction string = "undefined"
+)
+
+// Logger wraps the ethereum logger with specific information for swap logging
+type Logger struct {
```

Comment:
why was this renamed from `SwapLogger` to `Logger`?

Reply:
Redundant naming, the package is called swap and the field is SwapLogger, the linter complains of stutter code.

Reply:
that clears it up, thanks